app/src/main/java/com/example/marvelapp/domain/usecases/GetCharactersUseCaseImpl.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.marvelapp.domain.usecases
```

Comment:
this package should be usecase no usecases

Reply:
Done thx